src/main/scala/org/constellation/domain/redownload/RedownloadService.scala

```diff
@@ -132,6 +132,10 @@ class RedownloadService[F[_]: NonEmptyParallel](
         else m.updated(peer, existing + (proposal.value.height -> proposal))
       val ignored = InvertedMap(updated.mapValues(a => takeHighestUntilKey(a, getRemovalPoint(maxHeight(a)))))
       (ignored, ())
+    }.handleErrorWith { e =>
+      logger.error(e)(
+        s"Error persisting proposal of ${peer.hex} at height ${proposal.value.height} and hash ${proposal.value.hash}"
+      )
```

Comment:
raise error

Reply:
I will just remove that. I didn't realize that it swallows the error.